layouts/partials/custom_footer.html

```diff
@@ -0,0 +1,4 @@
+<!--
+To add custom footer section, please create
+`layouts/partials/custom_footer.html` in your Hugo directory.
```

Comment:
I'm thinking about the name... Is it `custom_footer` or just `extended_footer`? Maybe it's not a big deal, but in this case we don't override whole footer, but only extend it 🤔

Reply:
That's a good point. I think `extended_footer` makes more sense, I'll update the PR.

Reply:
I renamed it to `extended_footer`.